src/status_im2/contexts/shell/style.cljs

```diff
@@ -25,11 +25,48 @@
 
 ;; Home Stack
 (defn home-stack []
-  (let [{:keys [width height]} (constants/dimensions)]
+  (let [{:keys [width height]} (shell.constants/dimensions)]
     {:border-bottom-left-radius  20
      :border-bottom-right-radius 20
      :background-color           (colors/theme-colors colors/neutral-5 colors/neutral-95)
      :overflow                   :hidden
      :position                   :absolute
      :width                      width
-     :height                     (- height (constants/bottom-tabs-container-height))}))
+     :height                     (- height (shell.constants/bottom-tabs-container-height))}))
+
+;; Placeholder
+(defn placeholder-container [status-bar-height]
+  {:position            :absolute
+   :top                 (+ 112 status-bar-height)
+   :left                0
+   :right               0
+   :bottom              (shell.constants/bottom-tabs-container-height)
+   :align-items         :center
+   :accessibility-label :shell-placeholder-view})
+
+(def placeholder-image
+  {:margin-top 186
+   :width      120
+   :height     120})
+
+(def placeholder-title
+  {:size   :paragraph-1
+   :weight :semi-bold
+   :style  {:margin-top 20
+            :color      colors/white}})
+
+(def placeholder-subtitle
+  {:size   :paragraph-2
+   :weight :regular
+   :align  :center
+   :style  {:margin-top 4
+            :color      colors/white}})
+
+;; Shell
+(defn jump-to-text [status-bar-height]
+  {:size   :heading-1
+   :weight :semi-bold
+   :style  {:color         colors/white
```

Comment:
better to have only style in this ns, and do not mix with props, because it will be confusing

Reply:
probably we should add this to guidelines cc @ilmotta